packages/nuscr/nuscr.2.1.1/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis:
+  "A tool to manipulate and validate Scribble-style multiparty protocols"
+description:
+  "A toolkit to manipulate Scribble-style multiparty protocols, based on classical multiparty session type theory. The toolkit provides means to define global protocols, project to local protocols, convert local protocols to a CFSM representation, and generate OCaml code for protocol implementations."
+maintainer: ["Francisco Ferreira"]
+authors: [
+  "Francisco Ferreira" "Fangyi Zhou" "Simon Castellan" "Benito Echarren"
+]
+license: "GPL-3.0-or-later"
+homepage: "https://nuscr.dev/"
+doc: "https://nuscr.dev/nuscr/docs/"
+bug-reports: "https://github.com/nuscr/nuscr/issues"
+depends: [
+  "ocaml" {>= "4.08"}
```

Comment:
```suggestion
  "ocaml" {>= "4.10"}
```
`module _ = M` is a construct only available since OCaml 4.10
```
#=== ERROR while compiling nuscr.2.1.1 ========================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/nuscr/nuscr/releases/download/2.1.1/nuscr-2.1.1.tbz)
# path                 ~/.opam/4.08/.opam-switch/build/nuscr.2.1.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p nuscr -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/nuscr-8390-2dfee6.env
# output-file          ~/.opam/log/nuscr-8390-2dfee6.out
### output ###
# (cd _build/default && /home/opam/.opam/4.08/bin/menhir --explain -la 2 lib/syntaxtree/parser.mly --base lib/syntaxtree/parser --infer-write-query lib/syntaxtree/parser__mock.ml.mock)
# File "lib/syntaxtree/parser.mly", line 38, characters 7-15:
# Warning: the token RESERVED is unused.
# (cd _build/default && .ppx/9befcee2ef54134690458f3dcbb907c5/ppx.exe --cookie 'library-name="nuscrlib"' -o lib/utils/names.pp.ml --impl lib/utils/names.ml -corrected-suffix .ppx-corrected -diff-cmd - -dump-ast)
# File "lib/utils/names.ml", line 78, characters 7-8:
# 78 | module _ = struct
#             ^
# Error: Syntax error
# (cd _build/default && /home/opam/.opam/4.08/bin/menhir --explain -la 2 lib/syntaxtree/parser.mly --base lib/syntaxtree/parser --infer-read-reply lib/syntaxtree/parser__mock.mli.inferred)
# Built an LR(0) automaton with 752 states.
# The construction mode is pager.
# Built an LR(1) automaton with 752 states.
```